README.md

```diff
@@ -56,35 +153,49 @@ const spectral = new Spectral({ rulesets: [defaultRuleset()] });
 
 // run!
 console.log(spectral.run({ spec: 'oas2', target: myOAS }));
-
-//  [ {
-//   path: '$.responses',
-//   rule:
-//     { type: 'pattern',
-//       name: 'all-responses-must-be-numeric',
-//       path: '$..responses',
-//       enabled: true,
-//       description: 'reference components should all match regex ^[0-9]+',
-//       pattern: { property: '*', value: '^[0-9]+$' } },
-//     error:
-//       Error {
-//         operator: 'to be',
-//         expected: true,
-//         message: 'reference components should all match regex ^[0-9]+',
-//         showDiff: true,
-//         actual: false,
-//         stackStartFunction: [Function: assert],
-//         negate: false,
-//         assertion:
-//         Assertion {
-//           obj: false,
-//           anyOne: false,
-//           negate: false,
-//           params: [Object],
-//           onlyThis: undefined,
-//           light: false } } } ]
 ```
 
+Note: The existing OAS rulesets are opinionated. There might be some rules that you prefer to change. We encourage you to create your rulesets to fit your use case. We welcome contributions to the existing rulesets too!
+
+### Example Implementations
+
+- [Spectral Bot](https://github.com/tbarn/spectral-bot), a GitHub pull request bot that lints your repo's OAS document that uses the [Probot](https://probot.github.io) framework, built by [Taylor Barnett](https://github.com/tbarn)
+
+## FAQs
+
+**How is this different than [Ajv](https://github.com/epoberezkin/ajv)?**
+
+Ajv is a JSON Schema validator, not a linter. While Spectral runs your JSON objects through Ajv and returns the result to ensure you are using valid JSON, it goes a level beyond that and allows you to build a custom ruleset that lints your JSON objects.
```

Comment:
> While Spectral runs your JSON objects through Ajv and returns the result to ensure you are using valid JSON

Spectral does expose a `schema` function that you can use in your rules to validate all or part of the target object with JSON Schema (AJV is used under the hood). However, Spectral also provides a number of other functions and utilities that you can use to build up a linting ruleset that validates things JSON Schema is not suited for.

Reply:
I'll just add that.